rero_ils/modules/loans/api.py

```diff
@@ -605,6 +606,14 @@ def library_pid(self):
         """Get library PID regarding loan location."""
         return Location.get_record_by_pid(self.location_pid).library_pid
 
+    @property
+    def checkout_library_pid(self):
+        """Get the checkout library pid."""
+        checkout_location = Location.get_record_by_pid(
```

Comment:
In one line?

Reply:
Mmmm. I need to know if location is well loaded before asking `library_pid` attribute. 
Could you suggest a one line implementation ?